src/sentry/sentry_metrics/configuration.py

```diff
@@ -42,7 +42,7 @@ class MetricsIngestConfiguration:
     cardinality_limiter_cluster_options: Mapping[str, Any]
     cardinality_limiter_namespace: str
 
-    index_tag_values_option_name: Optional[str] = None
+    should_index_tag_values: bool = False
```

Comment:
small q, should this be `Optional[bool] = False`?

Reply:
I don't think it should, since the default is set to `False` and it should either be `True` or `False` and never None

Reply:
Nit: Though it does not change the behavior, it might be better to not have a default. Let each configuration explicitly set it following the general rule `Explicit is better than implicit`.

Reply:
Fixed, thank you!